interp/dumpglob.mli

```diff
@@ -19,11 +19,8 @@ type glob_output =
   | MultFiles                   (* one glob file per .v file *)
   | File of string              (* Single file for all coqc arguments *)
 
-(* Default "NoGlob" *)
-val set_glob_output : glob_output -> unit
-
-val pause : unit -> unit
-val continue : unit -> unit
+val push_output : glob_output -> unit
+val pop_output : unit -> unit
```

Comment:
Umm, I'm not so sure this API is better than the old one; push being used to stop the output looks confusing, in particular this is not a stack but more of a save / restore issue [I would expect push to _add_ one output to a set, not to override]

Reply:
Please add `ocamldoc` documentation to this functions, in particular as it is not clear what that the semantics of `push` do really mean `set`.